src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
+      IVariableDeclaration: x (Type: int) (OperationKind.VariableDeclaration)
+        Initializer: ILiteralExpression (Text: 3) (OperationKind.LiteralExpression, Type: int, Constant: 3)
+    IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+      IVariableDeclaration: r (Type: E) (OperationKind.VariableDeclaration)
+        Initializer: IBinaryOperatorExpression (BinaryOperationKind.EnumSubtract) (OperationKind.BinaryOperatorExpression, Type: E)
+            Left: ILocalReferenceExpression: x (OperationKind.LocalReferenceExpression, Type: int)
+            Right: ILocalReferenceExpression: e (OperationKind.LocalReferenceExpression, Type: E)
+    IExpressionStatement (OperationKind.ExpressionStatement)
+      IInvocationExpression (static void System.Console.Write(object)) (OperationKind.InvocationExpression, Type: void)
+        IArgument (Matching Parameter: value (Type: object)) (OperationKind.Argument)
+          IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: object)
+            ILocalReferenceExpression: r (OperationKind.LocalReferenceExpression, Type: E)
+    IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+      IVariableDeclaration: en (Type: E?) (OperationKind.VariableDeclaration)
+        Initializer: IConversionExpression (ConversionKind.CSharp, Implicit) (OperationKind.ConversionExpression, Type: E?)
+            IFieldReferenceExpression: E.Two (Static) (OperationKind.FieldReferenceExpression, Type: E, Constant: 2)
+    IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+      IVariableDeclaration: xn (Type: int?) (OperationKind.VariableDeclaration)
+        Initializer: IConversionExpression (ConversionKind.CSharp, Implicit) (OperationKind.ConversionExpression, Type: int?)
+            ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: int, Constant: 2)
+    IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+      IVariableDeclaration: rn (Type: E?) (OperationKind.VariableDeclaration)
+        Initializer: IBinaryOperatorExpression (BinaryOperationKind.EnumSubtract) (OperationKind.BinaryOperatorExpression, Type: E?)
+            Left: ILocalReferenceExpression: xn (OperationKind.LocalReferenceExpression, Type: int?)
+            Right: ILocalReferenceExpression: en (OperationKind.LocalReferenceExpression, Type: E?)
+    IExpressionStatement (OperationKind.ExpressionStatement)
+      IInvocationExpression (static void System.Console.Write(object)) (OperationKind.InvocationExpression, Type: void)
+        IArgument (Matching Parameter: value (Type: object)) (OperationKind.Argument)
+          IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: object)
+            ILocalReferenceExpression: rn (OperationKind.LocalReferenceExpression, Type: E?)",
+"Main");
```

Comment:
Sorry I didn't realize the PR is just for commenting on the text dump, you can ignore my other comments for now :)
The dump looks fine (at least for this specific test, it's easy to follow). This approach is great. I assume this means there's no need to craft more test code since we can reuse existing ones which should cover all grounds?